<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortSetting</class>
 <widget class="QDialog" name="SerialPortSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>29</y>
     <width>140</width>
     <height>221</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>221</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>221</height>
    </size>
   </property>
   <property name="title">
    <string>Select Serial Port</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="serialPortInfoListBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="descrptionlabel">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="manufacturerlabel">
        <property name="text">
         <string>Manufacturer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="serialnumberlabel">
        <property name="text">
         <string>Serial number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="locationlabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vidlabel">
        <property name="text">
         <string>Vendor ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pidlabel">
        <property name="text">
         <string>Product ID:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>179</width>
     <height>221</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>221</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>221</height>
    </size>
   </property>
   <property name="title">
    <string>Select Parameters</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>BaudRate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudrateBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Data bits:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="databitsBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Parity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="parityBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Stop bits:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stopbitBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Flow control:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="flowcontrolBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>362</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>198</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
